market-test/src/test/java/com/axonivy/market/test/ValidateRepoTest.java

```diff
@@ -17,11 +17,43 @@
 
 import javax.imageio.ImageIO;
 
-import org.json.JSONObject;
 import org.junit.jupiter.api.Test;
 
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.networknt.schema.JsonSchema;
+import com.networknt.schema.JsonSchemaFactory;
+import com.networknt.schema.SpecVersion.VersionFlag;
+import com.networknt.schema.ValidationMessage;
+
 class ValidateRepoTest
 {
+  private static final ObjectMapper MAPPER = new ObjectMapper();
+
+  @Test
+  void everyProductHasAValidMetaJson_schemaDriven()
+  {
+    assertThat(productDirs()).allSatisfy(path -> assertValidMetaJsonSchema(path));
+  }
+
+  private void assertValidMetaJsonSchema(Path path)
+  {
+    var metaPath = path.resolve("meta.json");
+    var metaImpl = load(metaPath);
+    URI metaSchemaUri = URI.create("https://jenkins.ivyteam.io/job/core_json-schema/job/cc23.marketSchema/lastSuccessfulBuild/artifact/workspace/ch.ivyteam.ivy.market.schema/target/schema/market/0.0.1/meta.json");
+    JsonSchema metaSchema = JsonSchemaFactory.getInstance(VersionFlag.V201909).getSchema(metaSchemaUri);
+    List<String> messages = metaSchema.validate(metaImpl).stream().map(ValidationMessage::getMessage).toList();
```

Comment:
lists all validation errors in a single meta.json ... unlike our old homebrewn approach that only listed the first problem 
![newTests](https://github.com/axonivy-market/market/assets/15085693/97cd30f0-ee72-4899-8ff8-b2654fb112b3)
![oldTest](https://github.com/axonivy-market/market/assets/15085693/db4e0390-6d94-45a7-bde2-f78ca3d75b39)